releases/release-1.18/release_team.md

```diff
@@ -3,7 +3,7 @@
 | **Role** | **Name** (**GitHub / Slack ID**)  | **Shadow Name(s) (GitHub / Slack ID)** |
 | ------ | ------ | ------ |
 | Lead | Jorge Alarcon ([@alejandrox1](https://github.com/alejandrox1) / Slack: `@alejandrox1`) | Damini Satya ([@daminisatya](https://github.com/daminisatya) / Slack: `@Damini Satya`), Bob Killen ([@mrbobbytables](https://github.com/mrbobbytables) / Slack: `@mrbobbytables`) |
-| Enhancements | Jeremy Rickard ([jeremyrickard](https://github.com/jeremyrickard) / Slack: `@jerickar`) |  |
+| Enhancements | Jeremy Rickard ([jeremyrickard](https://github.com/jeremyrickard) / Slack: `@jerickar`) | Nabarun Pal ([palnabarun](https://github.com/palnabarun) / Slack: `@palnabarun`), Kirsten Garrison ( [kikisdeliveryservice](https://github.com/kikisdeliveryservice) / Slack: `@oikiki`), John Belamaric ([johnbelamaric](https://github.com/johnbelamaric) / Slack: `@johnbelamaric`), Heba Elayoty ([helayoty](https://github.com/helayoty) / Slack: `@Heba`)  |
```

Comment:
```suggestion
| Enhancements | Jeremy Rickard ([@jeremyrickard](https://github.com/jeremyrickard) / Slack: `@jerickar`) | Nabarun Pal ([@palnabarun](https://github.com/palnabarun) / Slack: `@palnabarun`), Kirsten Garrison ( [@kikisdeliveryservice](https://github.com/kikisdeliveryservice) / Slack: `@oikiki`), John Belamaric ([@johnbelamaric](https://github.com/johnbelamaric) / Slack: `@johnbelamaric`), Heba Elayoty ([@helayoty](https://github.com/helayoty) / Slack: `@Heba`)  |
```

Reply:
OH HOW DID I MISS THAT 🤦‍♂

Reply:
:mag: :mag_right: :mag: :mag_right: :mag: :mag_right: :mag: :mag_right: :mag: :mag_right: :mag: :mag_right: